### PR TITLE
Renames ranks

### PR DESCRIPTION
**Pull request by captainwiseguy**
## About The Pull Request
fixes ranks so theyre not stupid

## Why It's Good For The Game
better ranks 

## Changelog
:cl:
config: changed ranks
/:cl:

### CODE REVIEW

File: config/ranks/military.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
+Head of Security=LT
+Chief Engineer=LTCDR
```
Comment (covertcorvid) [inline config/ranks/military.txt]: WO specialty: Engineering/Nuclear Power Technician
This isn't even the right abbreviation for lieutenant commander.

File: config/ranks/military.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
+Head of Security=LT
```
Comment (covertcorvid) [inline config/ranks/military.txt]: WO specialty: Security Technician
Why would the HOS be lower-ranking than the rest of the heads?

File: config/ranks/military.txt [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+Chief Engineer=LTCDR
+Research Director=LT
```
Comment (covertcorvid) [inline config/ranks/military.txt]: WO specialty: electronics technician, information systems technician, EOD technician

File: config/ranks/military.txt [hunk 4]
```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
```
Comment (covertcorvid) [inline config/ranks/military.txt]: Honestly I'd at least bring everything down by one and make the captain a commander and the exec a lieutenant commander

File: config/ranks/military.txt [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
+Head of Security=LT
+Chief Engineer=LTCDR
+Research Director=LT
+Chief Medical Officer=LTCDR
```
Comment (covertcorvid) [inline config/ranks/military.txt]: Okay I'll buy that doctors are usually officers but you still need to fix the abbreviation.

**Comment by Kmc2000** [comment]
I feel like the ranks system isn't that bad, and I'm not really open to changing it without a large amount of people complaining. I'd prefer that this was made into an alternative config file so that if (or when) we decided to change the ranks about, the change would just require us to shift the file that's loaded.

**Comment by Vasily2013** [comment]
Also your abreviation of lieutenant commander is wrong, as is the all caps formatting.

**Comment by Vasily2013** [comment]
Altered title to more properly represent what the PR does, as ranks were not previously broken.

**Comment by francinum** [comment]
The ranks don't need to be adjusted. If you'de like to suggest a new set of ranks please create them in a new file.

**Pull request closed without merging**